risc0/zkvm/src/lib.rs

```diff
@@ -12,11 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![doc = include_str!("../README.md")]
```

Comment:
I see what's going on here... A future improvement would be to figure out a way to strip off the `<!-- cargo-rdme ... -->`. I think this is good for the time-being though.

Reply:
It's kinda how `cargo-rdme` works, and I'm not sure it's an issue?

Reply:
Oh I had a misunderstanding! Please disregard this comment!